.changes/v2.20.0/550-improvements.md

```diff
@@ -0,0 +1,2 @@
+* Added support for using subnet prefix length while creating vApp networks. [GH-550]
```

Comment:
Ah! Nit, but we don't use dots at the end of changelog entries. You can check here:
https://github.com/vmware/go-vcloud-director/blob/main/CHANGELOG.md

Reply:
Got it, removed